snu-amp19-team1/drunkare-tizen
# Transfer sensor data in JSON format using curl

### Test
- Set `DURATION` to set the transfer period (default : 60 sec)
- Check `drunkare` log

### Issue
- If you set the `DURATION` small (e.g. < 10 sec), there is some queuing latency caused by curl instruction
